build.sbt

```diff
@@ -1147,7 +1167,11 @@ lazy val runtime = (project in file("engine/runtime"))
       "org.scalatest"      %% "scalatest"             % scalatestVersion  % Test,
       "org.graalvm.truffle" % "truffle-api"           % graalVersion      % Benchmark,
       "org.typelevel"      %% "cats-core"             % catsVersion,
-      "eu.timepit"         %% "refined"               % refinedVersion
+      "eu.timepit"         %% "refined"               % refinedVersion,
+      // This dependency is needed only so that developers don't download Frgaal manually.
+      // Sadly it cannot be placed under plugins either because meta dependencies are not easily
+      // accessible from the non-meta build definition.
+      FrgaalJavaCompiler.frgaal
```

Comment:
Does this mean that we are going to be packaging Frgaal with our runtime?

Reply:
No, the classifier of that dependency is `provided`